modules/network-operator-cr.adoc

```diff
@@ -0,0 +1,142 @@
+configuration. If you do not specify the parameter values, the \
+Network Operator applies the default parameter values.]
+
+:kube-proxy-ip: pass:q[The IP address for `kube-proxy` to serve on. \
+The default value is `0.0.0.0`.]
+
+:kube-proxy-refresh: pass:q[The refresh period for `iptables` rules. The default \
+value is `30s`. Valid suffixes include `s`, `m`, and `h` and are described in \
+the link:https://golang.org/pkg/time/#ParseDuration[Go time package] \
+documentation.]
+
+:iptables-min-sync-period: pass:q[The minimum duration before refreshing `iptables` \
+rules. This parameter ensures that the refresh does not happen too frequently. \
+Valid suffixes include `s`, `m`, and `h` and are described in the \
+link:https://golang.org/pkg/time/#ParseDuration[Go time package]]
+
+// Begin module
+
+[id="network-networkoperator-default-cr-{context}"]
+= Network Operator Custom Resource (CR)
+
+The network configuration in an `Network.operator.openshift.io` Custom Resource
+(CR) stores the configuration settings for the Network Operator.
+
+ifdef::install[]
+The following parameters are set in the `install-config.yaml` file:
+
+* `clusterNetwork`
+* `serviceNetwork`
+* `networkType`
+endif::install[]
+
+The following CR displays the default configuration for the Network Operator and
+explains both the parameters you can configure and valid parameter values:
+
+.Network Operator CR
+[source,yaml]
+ifdef::install[]
+----
+apiVersion: operator.openshift.io/v1
+kind: Network
+spec:
+  clusterNetwork: <1>
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  serviceNetwork: 172.30.0.0/16 <1>
+  defaultNetwork:
+    type: OpenShiftSDN <1>
+    openshiftSDNConfig: <2>
+      mode: NetworkPolicy <3>
+      mtu: 1450 <4>
+      vxlanPort: 4789 <5>
+  kubeProxyConfig: <6>
+    bindAddress: 0.0.0.0 <7>
+    iptablesSyncPeriod: 30s <8>
+    proxyArguments:
+      iptables-min-sync-period: <9>
+      - 30s
+----
+<1> Specified in the `install-config.yaml` file.
+
+<2> Specify only if you want to override part of the {product-title} SDN
+configuration.
+
+<3> Configures the isolation mode for `OpenShiftSDN`. The allowed values are
+`Multitenant`, `Subnet`, or `NetworkPolicy`. The default value is
+`NetworkPolicy`.
+
+<4> MTU for the VXLAN overlay network. This is normally configured
+automatically, but if the nodes in your cluster do not all use the same MTU,
+then you must set this explicitly to 50 less than the smallest node MTU value.
+
+<5> The port to use for all VXLAN packets. The default value is 4789. If you are
+running in a virtualized environment with existing nodes that are part of
+another VXLAN network then you may need to change this.
+
+<6> {kube-proxy}
+<7> {kube-proxy-ip}
+<8> {kube-proxy-refresh}
+<9> {iptables-min-sync-period}
+endif::install[]
+
+ifdef::not-install[]
+----
+apiVersion: operator.openshift.io/v1
+kind: Network
+spec:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14 <1>
+    hostPrefix: 23 <1>
```

Comment:
Should have some example with multiple cidrs. Also, the examples can use different values for the fields to show what can be done.